Adds `hasLink` test for examples; update ~37 descriptions to pass (#474)
mapbox/mapbox-gl-js-docs: docs/pages/example/updating-choropleth.md

- "import html from './updating-choropleth.html';"
---

This map uses 2014 U.S. Census data to create two layers. One layer contains a choropleth visualization for _state populations_, and one layer contains a choropleth visualization for _county populations_. 

> The example defines a zoom level at which each layer and its respective legend should appear or disappear using the `minzoom` and `minzoom` [layer properties](/mapbox-gl-js/style-spec/layers/).

Author: 293a686 7ccdd19 👍

> You can use [`getZoom`](/mapbox-gl-js/api/map/#map#getzoom) with the [`MapBoxZoomEvent`](/mapbox-gl-js/api/events/#mapboxzoomevent) event type to get zoom values to control the appearance of other elements. 

Author: cf0ef1a

- "import html from './updating-choropleth.html';"
---

> This map uses 2014 U.S. Census data to create two layers, one containing a choropleth visualization for _state populations_, and one containing a choropleth visualization for _county populations_. 
> 
> Then, it defines a zoom level at which each layer and its respective legend should appear or disappear.
> 
> You can use the `minzoom` and `minzoom` [layer properties](/mapbox-gl-js/style-spec/layers/) to define the zoom levels at which a map layer should appear or disappear.
> 
> You can use [`getZoom`](/mapbox-gl-js/api/map/#map#getzoom) with the [`MapBoxZoomEvent`](/mapbox-gl-js/api/events/#mapboxzoomevent) event type to get zoom values to control the appearance of other elements. 

Reviewer: ```suggestion
This map uses 2014 U.S. Census data to create two layers. One layer contains a choropleth visualization for _state populations_, and one layer contains a choropleth visualization for _county populations_. 

The example defines a zoom level at which each layer and its respective legend should appear or disappear using the `minzoom` and `minzoom` [layer properties](/mapbox-gl-js/style-spec/layers/).

You can use [`getZoom`](/mapbox-gl-js/api/map/#map#getzoom) with the [`MapBoxZoomEvent`](/mapbox-gl-js/api/events/#mapboxzoomevent) event type to get zoom values to control the appearance of other elements. 
```
